components/deploy/DeploymentInfoOverlay.tsx

```
@@ -0,0 +1,42 @@
+
+export function DeploymentInfoOverlay({ result, error, isLoading, onClose }: DeploymentInfoOverlayProps) {
+  if (!result && !error && !isLoading) return null;
+
+  return (
+    <div className="fixed inset-0 z-50 flex items-center justify-center bg-black/40">
+      <div className="bg-white dark:bg-zinc-900 rounded-lg shadow-xl p-6 min-w-[320px] max-w-[90vw] relative">
+        <button
+          className="absolute top-2 right-2 text-zinc-400 hover:text-zinc-900 dark:hover:text-white text-xl"
+          onClick={onClose}
+          aria-label="Close"
+        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)